package.json

```diff
@@ -47,8 +49,8 @@
   "dependencies": {
     "algoliasearch-helper": "^3.1.0",
     "instantsearch.js": "^4.20.0",
-    "vue-demi": "npm:@eunjae-lee/vue-demi@0.9.3",
-    "mitt": "^2.1.0"
+    "mitt": "^2.1.0",
+    "vue-demi": "npm:@eunjae-lee/vue-demi@0.10.2"
```

Comment:
is vue-demi a runtime dependency?

Reply:
Yes. I updated the description of this PR to have more info on vue-demi.